frontend/libs/ui-modules/src/modules/contacts/components/CompaniesInline.tsx

```
@@ -0,0 +1,72 @@
+    if (companyDetail) {
+      const existingCompaniesMap = new Map(
+        companies.map((company) => [company._id, company]),
+      );
+      const newCompanies = companies.filter(
+        (company) => !existingCompaniesMap.has(company._id),
+      );
+      if (newCompanies.length > 0) {
+        updateCompanies?.([...companies, ...newCompanies]);
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)